web/src/Components/Dialog.elm

```diff
@@ -44,15 +45,22 @@ dialog model =
                 --                    memberInviteDialogV model
                 NotFoundD ->
                     el Error [] (text "Not Found")
+
+        closeButton =
+            if model.voteConfirmStatus == Processing || model.voteConfirmStatus == Validating then
```

Comment:
works for now but need to refactor this to be a flag provided to dialog in future